pkg/cmd/importcmd/import.go

```diff
@@ -223,7 +223,9 @@ func (o *ImportOptions) AddImportFlags(cmd *cobra.Command, createProject bool) {
 	cmd.Flags().BoolVarP(&o.Destination.JenkinsX.Enabled, "jx", "", false, "if you want to default to importing this project into Jenkins X instead of a Jenkins server if you have a mixed Jenkins X and Jenkins cluster")
 	cmd.Flags().StringVarP(&o.Destination.JenkinsfileRunner.Image, "jenkinsfilerunner", "", "", "if you want to import into Jenkins X with Jenkinsfilerunner this argument lets you specify the container image to use")
 	cmd.Flags().StringVar(&o.ServiceAccount, "service-account", "tekton-bot", "The Kubernetes ServiceAccount to use to run the initial pipeline")
+	cmd.Flags().StringVar(&o.SchedulerName, "scheduler-name", "in-repo", "Change schedulerName, More info in https://jenkins-x.io/v3/develop/faq/config/repos/#how-do-i-customise-a-scheduler")
```

Comment:
Where will this be set in the code? So, if I do `jx project import --scheduler xyz` will it change it?

Reply:
already changed to scheduler . This option will change the in the dev environment jx/scheduler. jx/gitops/source-config.yaml。The specific implementation in pkg/cmd/importcmd/source_config_pr.go line 74